answerfile.yml

```diff
@@ -38,96 +38,78 @@ uplink_profiles:
 #     policy: FAILOVER_ORDER
 #   transport_vlan: 0
 
-transport_nodes:
+host_transport_nodes:
 - display_name: TN1
   host_switches:
   - host_switch_profiles:
     - name: uplinkProfile1
       type: UplinkHostSwitchProfile
     host_switch_name: hostswitch1
-    # pnics:
-    # - device_name: "vmnic1"
-    #   uplink_name: "uplink-1"
+    host_switch_mode: STANDARD
```

Comment:
Dont we need host_switch_type: VDS/NVDS too? Perhaps we need to take a step back and look at all the examples in build_topology.yml + answerfile.yml. Is it being tested? If not, perhaps we should mention the last version it worked it.